src/utils/datapackage.js

```diff
@@ -21,3 +21,21 @@ export async function fetchDataPackageAndData(dataPackageIdentifier) {
   return dp
 }
 
+export async function fetchDataPackageOnly(dataPackageIdentifier) {
+  return await new Datapackage(dataPackageIdentifier)
+}
+
+export async function fetchDataOnly(resource) {
```

Comment:
When do we use this function?

I would suggest this is too small a commit. You are making a change without any clear need or context.

Also what about the tests -- surely this would break tests?